Home screen update

ChrisRu opened the pull request:
This updates the dependencies and fixes the homepage to show live now correctly.

ChrisRu committed:
Update packages

ChrisRu committed:
Fix live now on homepage

olidotjpeg commented on src/pages/HomePage.tsx:
```diff
@@ -182,51 +187,55 @@ function HomePage({event}: IProps & RouteComponentProps) {
-              {/*    Attendee Guide*/}
-              {/*  </HeaderLinks>*/}
-              {/*</HeaderMetaList>*/}
+              <HeaderMetaList>
```
We shouldn't show these atm, since they are linking to old outdated stuff.

ChrisRu replied:
Ah yes, I temporarily added it cuz it somehow broke the home page header slidey thingy.

ChrisRu committed:
Fix date in sidebar and remove metalist in home page header

ChrisRu commented:
Removed it again. Also added some fixes for static dates in the menubar and fixed the home page gesture. Bit of a pain to set the height manually but there's not much of a choice.
